share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```
@@ -375,6 +375,9 @@ function(ExternalZephyrProject_Add)
   if(DEFINED ZBUILD_APP_TYPE)
     set(image_default "${CMAKE_SOURCE_DIR}/image_configurations/${ZBUILD_APP_TYPE}_image_default.cmake")
     set_target_properties(${ZBUILD_APPLICATION} PROPERTIES IMAGE_CONF_SCRIPT ${image_default})
+  else()
+    set(image_default "${CMAKE_SOURCE_DIR}/image_configurations/ALL_image_default.cmake")
+    set_target_properties(${ZBUILD_APPLICATION} PROPERTIES IMAGE_CONF_SCRIPT ${image_default})
   endif()
```

Just append for all images:
```suggestion
  endif()
  set(image_default "${CMAKE_SOURCE_DIR}/image_configurations/ALL_image_default.cmake")
  set_property(TARGET ${ZBUILD_APPLICATION} APPEND PROPERTY IMAGE_CONF_SCRIPT ${image_default})
```